backend/cmd/headlamp.go

```diff
@@ -824,30 +823,6 @@ func createHeadlampHandler(config *HeadlampConfig) http.Handler {
 	return r
 }
 
-func parseClusterAndToken(r *http.Request) (string, string) {
-	cluster := ""
-	re := regexp.MustCompile(`^/clusters/([^/]+)/.*`)
-	urlString := r.URL.RequestURI()
-
-	matches := re.FindStringSubmatch(urlString)
-	if len(matches) > 1 {
-		cluster = matches[1]
-	}
-
-	// Try Authorization header first (for backward compatibility)
-	token := r.Header.Get("Authorization")
-	token = strings.TrimPrefix(token, "Bearer ")
-
-	// If no auth header, try cookie
```

Comment:
the logic of fetching token from the cookie was removed in this PR. Could you check this @skoeva ? I don't think this was intentional and I suspect this is causing issues with oidc token refresh